src/utils/createFile.cpp

```diff
@@ -5,8 +5,10 @@
 #include "logger/logging.hpp"
 
 void createFile(const std::string &filename, const std::string &fileContent) {
+
 	coloredLog("Creating file: ", filename, GREEN);
 //	coloredLog("File content: ", fileContent, GREEN);
+
 	std::ofstream file;
 	file.open(("uploadedFiles/" + filename).c_str());
 	file << fileContent;
```

Comment:
secure open ?

using this code :

std::ofstream file("filename");
if (file.is_open())....